packages/processors/test/strategy/donationVotingMerkleDistributionDirectTransfer/handlers/timestampsUpdated.handler.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { beforeEach, describe, expect, it, vi } from "vitest";
+
+import { IRoundReadRepository, PartialRound, Round } from "@grants-stack-indexer/repository";
+import { ChainId, DeepPartial, mergeDeep, ProcessorEvent } from "@grants-stack-indexer/shared";
+
+import { RoundNotFound } from "../../../../src/exceptions/index.js";
+import { DVMDTimestampsUpdatedHandler } from "../../../../src/strategy/donationVotingMerkleDistributionDirectTransfer/handlers/index.js";
+
+function createMockEvent(
+    overrides: DeepPartial<
+        ProcessorEvent<"Strategy", "TimestampsUpdatedWithRegistrationAndAllocation">
+    > = {},
+): ProcessorEvent<"Strategy", "TimestampsUpdatedWithRegistrationAndAllocation"> {
+    const defaultEvent: ProcessorEvent<
+        "Strategy",
+        "TimestampsUpdatedWithRegistrationAndAllocation"
+    > = {
+        params: {
+            registrationStartTime: 1000000000n,
+            registrationEndTime: 1000086400n, // +1 day
+            allocationStartTime: 1000172800n, // +2 days
+            allocationEndTime: 1000259200n, // +3 days
+            sender: "0xcBf407C33d68a55CB594Ffc8f4fD1416Bba39DA5",
+        },
+        eventName: "TimestampsUpdatedWithRegistrationAndAllocation",
+        srcAddress: "0x1234567890123456789012345678901234567890",
+        blockNumber: 12345,
+        blockTimestamp: 1000000000,
+        chainId: 10 as ChainId,
+        contractName: "Strategy",
+        logIndex: 1,
+        transactionFields: {
+            hash: "0xd2352acdcd59e312370831ea927d51a1917654697a72434cd905a60897a5bb8b",
+            transactionIndex: 6,
+            from: "0xcBf407C33d68a55CB594Ffc8f4fD1416Bba39DA5",
+        },
+        strategyId: "0x9fa6890423649187b1f0e8bf4265f0305ce99523c3d11aa36b35a54617bb0ec0",
+    };
+
+    return mergeDeep(defaultEvent, overrides);
+}
+
+describe("DVMDTimestampsUpdatedHandler", () => {
+    let handler: DVMDTimestampsUpdatedHandler;
+    let mockRoundRepository: IRoundReadRepository;
+    let mockEvent: ProcessorEvent<"Strategy", "TimestampsUpdatedWithRegistrationAndAllocation">;
+    const chainId = 10 as ChainId;
+
+    beforeEach(() => {
+        mockRoundRepository = {
+            getRoundByStrategyAddress: vi.fn(),
+        } as unknown as IRoundReadRepository;
+    });
+
+    it("handle a valid timestamps update event", async () => {
+        const timestamps = {
+            registrationStartTime: 1704067200n, // 2024-01-01 00:00:00
+            registrationEndTime: 1704153600n, // 2024-01-02 00:00:00
+            allocationStartTime: 1704240000n, // 2024-01-03 00:00:00
+            allocationEndTime: 1704326400n, // 2024-01-04 00:00:00
+        };
```

Comment:
may we check if envio is actually returning this params of the event as string or number ?

Reply:
you're right here, it's string. should we treat as string or have the indexer-client parse it and coerce to bigint?